### ISSUE TITLE
Unable to extract table data with has <th> and <td> in it

### ISSUE THREAD

**Issue by Bobby2210** [yuanxu-li/html-table-extractor#13]
<table id="productDetails_techSpec_section_1" class="a-keyvalue prodDetTable" role="presentation">
  | <tr>
  | <th class="a-color-secondary a-size-base prodDetSectionEntry">
  | Brand
  | </th>

  | <td class="a-size-base">
  | AmazonBasics
  | </td>
  | </tr>
</table>

Then above structure is not getting extracted.

**Comment by yuanxu-li** [comment]
I think this is the right format. Let me try it out
```
<table id="productDetails_techSpec_section_1" class="a-keyvalue prodDetTable" role="presentation">
<tr>
<th class="a-color-secondary a-size-base prodDetSectionEntry">
Brand
</th>
<td class="a-size-base">
AmazonBasics
</td>
</tr>
</table>
```

**Comment by yuanxu-li** [comment]
@Bobby2210 
Hi, it is fixed in the most recent version 1.4.0.

Thank you for your support :)
